app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -28,10 +28,12 @@
 
           <ul>
             <li>
-                <a class="marketing-link marketing-link-ios" href="{{ appStoreLinkWeb }}" target="_blank"><img src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS.{{/t}}"></a>
+                <a class="marketing-link marketing-link-android" href="https://play.google.com/store/apps/details?id=org.mozilla.firefox&amp;referrer=utm_source%3Dmozilla%26utm_medium%3DReferral%26utm_campaign%3Dsync-desktop-snippet"><img width="129" height="45" src="{{ playStoreImage }}" alt="{{#t}}Download Firefox for Android.{{/t}}"></a>
```

Comment:
iOS link was shortened because it wouldn't be available at merge/push time. We put it in an experiment so we could update the link without doing a full push.